lte/gateway/configs/gateway.mconfig

```diff
@@ -39,7 +39,8 @@
       "ipv6DnsAddress": "2001:4860:4860:0:0:0:0:8888",
       "natEnabled": true,
       "congestionControlEnabled": true,
-      "enableConvergedCore": false
+      "enableConvergedCore": false,
```

Comment:
add entry for `defaultSliceDifferentiator` here as well

Reply:
Comments addressed.